langchain/src/document_loaders/web/browserbase.ts

```diff
@@ -0,0 +1,83 @@
+import { Document } from "@langchain/core/documents";
+import type { BrowserbaseLoadOptions } from "@browserbasehq/sdk";
+import { BaseDocumentLoader } from "../base.js";
+import type { DocumentLoader } from "../base.js";
+
+type BrowserbaseLoaderOptions = BrowserbaseLoadOptions & {
+  apiKey?: string;
+};
+
+/**
+ * Load pre-rendered web pages using a headless browser hosted on Browserbase.
+ *
+ * Depends on `@browserbasehq/sdk` package.
+ * Get your API key from https://browserbase.com
+ *
+ * @param {string[]} urls - The URLs of the web pages to load.
+ * @param {BrowserbaseLoaderOptions} [options] - Browserbase client options.
+ */
```

Comment:
Adding `@example`'s in the top level JSDocs are great too! Not required, but they help out the end user a lot, and make your API ref page look better. See [here](https://api.js.langchain.com/functions/langchain_chains_retrieval.createRetrievalChain.html#:~:text=and%20answer%20keys.-,Example,-//%20yarn%20add%20langchain) for an example